### PR TITLE
Add new Testing Instructions section to PR template.

### PR DESCRIPTION
**Pull request by alexstine**
## Description

This adds a new section called Testing Instructions to the PR template. That way contributors can better understand exactly how to test a PR by following simple steps.

## How has this been tested?

GitHub template change.

## Types of changes

Workflow modification.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] I've tested my changes with keyboard and screen readers. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR (please manually search all `*.native.js` files for terms that need renaming or removal). 
- [x] I've updated related schemas if appropriate.

### CODE REVIEW

**Comment by noisysocks** [comment]
We already have "How has this been tested" which is supposed to cover this. I like the copy that you wrote better, though. How about we delete "How have this been tested" and keep "Testing instructions"?

**Comment by alexstine** [comment]
@noisysocks Sounds good. Updated.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -5,10 +5,14 @@
 ## Description
 <!-- Please describe what you have changed or added -->
 
-## How has this been tested?
+## Testing Instructions
 <!-- Please describe in detail how you tested your changes. -->
 <!-- Include details of your testing environment, tests ran to see how -->
 <!-- your change affects other areas of the code, etc. -->
```
Comment (talldan) [inline .github/PULL_REQUEST_TEMPLATE.md]: I think we could also delete this text in favour of the new text, which is much better in my opinion. What do you think?

Comment (alexstine) [reply .github/PULL_REQUEST_TEMPLATE.md]: Done.

**Review by talldan — APPROVED** [review]
I think this is a good improvement, so giving it the thumbs up 👍

**Review by getdave — APPROVED** [review]
I'm all for this. I still find too many PRs where the author describes that _they_ have tested it rather than how the reviewer should test it.

Thanks for the PR!

**Review by Mamaduka — APPROVED** [review]
It is a good improvement. Let's merge it :shipit:

**Pull request merged**